test/util.ts

```diff
@@ -10,6 +11,27 @@ export function mocked<T>(module: T): jest.Mocked<T> {
   return module as never;
 }
 
+/**
+ * Partially mock a module, providing an object with explicit mocks
+ * @param moduleName The module to mock
+ * @param overrides An object containing the mocks
+ * @example
+ * jest.mock('../../util/exec/docker/index', () =>
+ *   require('../../../test/util').mockPartial('../../util/exec/docker/index', {
+ *     removeDanglingContainers: jest.fn(),
+ *   })
+ * );
+ */
+export function mockPartial(moduleName: string, overrides?: object): unknown {
```

Comment:
Wouldn't it be better to pass the repo relative path?

Reply:
The path relative to `test/utils.ts` and the test that actually uses this method are different though.